build/.eslintrc.yaml

```diff
@@ -1,3 +1,11 @@
+extends: '@wfcd/eslint-config/esm'
+parserOptions:
+  babelOptions:
+    plugins:
+      - '@babel/plugin-syntax-import-assertions'
```

Comment:
yeah that's why i already sent a pr upstream

Reply:
i'm pretty sure this requires node 18 (for lts), as well, this is something we'd discussed on wfcd/warframe-worldstate-parser#354

Reply:
Yes it was after but i always test myself and it worked, but it may not  work depending on the version of node 16

Reply:
https://github.com/nodejs/node/pull/41736

Reply:
yeah, that's why i'd written the index.mjs how it's written for importing json in the old compatible way.

Reply:
that's another way to load json but for the future the json loading will be with assertions, i'm just future proofing and since we use at least v16 there shouldn't be any problem with it.

Reply:
the future "good way" that was supposed to only be going into v18 of lts and was in unstable until a few feature versions on LTS ago... i don't like using experiments when  they're still in experimental on 16

![image](https://user-images.githubusercontent.com/7128721/204204429-d6aab748-7707-477f-89f3-fa8a4b03a15e.png)

as i mentioned in WFCD/warframe-worldstate-parser#354, i'm fine using them when they're in stable state on those LTS versions, but not until then